### ISSUE TITLE
76 - Update/draw eat exceptions 

### ISSUE THREAD

**Issue by jlnr** [gosu/gosu#75]
<pre>
(Reposted from the forums, since I'm fairly certain this is a bug)

Using Ruby 1.9 with the precompiled Windows gem. Here's a minimal example
that shows my problem:

require 'rubygems'
require 'gosu'

class GameWindow &lt; Gosu::Window
  def initialize
    super(640, 480, false)
  end

  def update
    raise 'this exception gets eaten'
  end

  def draw
    raise 'and so does this one'
  end
end

window = GameWindow.new
window.show

Basically, any exception raised inside the update or draw methods is
silently &quot;eaten&quot;, and the method ends at that point. &quot;exit&quot; and &quot;abort&quot;
don't work either. I can't imagine this is by design, since it makes
debugging harder (you can work around it, but that's assuming you manage to
figure out the problem in the first place).
</pre></br>Original link: http://code.google.com/p/gosu/issues/detail?id=76
